### PR TITLE
Migrate glide to dep

### PR DESCRIPTION
**Pull request by jackchuka**
Trying migrating glide to dep.

## To reproduce
```bash
$ dep init ## will create Gopkg.(toml|lock)
## modify Gopkg.toml to include revisions
$ rm glide.lock glide.yaml ## remove glide files
## update README and Makefile
$ make bundle
```

## Check locked revisions
```bash
$ cat glide.lock | grep version
  version: bbd5bb678321a0d6e58f1099321dfa73391c1b6f
  version: 4b86f9c0ead51cc410d05655596e30f281ed9071
  version: ab9f90f16caab828afda479fd34bfbbbba2efcee
  version: 901cec093d58ca36652e8d7906e4bd3e32793801
  version: 1589c179b764ddceb204a439d9bf366f04c55f9b
  version: 18a02ba4a312f95da08ff4cfc0055750ce50ae9e
  version: 3b8db5e93c4c02efbc313e17b2e796b0914a01fb
  version: 12592ca48efc73057c0edcdddc2a279b37c2b27b
  version: 4971afdc2f162e82d185353533d3cf16188a9f4e
  version: 04cdfd42973bb9c8589fd6a731800cf222fde1a9
  version: d8ed2627bdf02c080bf22230dbb337003b7aba2d

$ cat Gopkg.lock | grep revision
  revision = "bbd5bb678321a0d6e58f1099321dfa73391c1b6f"
  revision = "1589c179b764ddceb204a439d9bf366f04c55f9b"
  revision = "4b86f9c0ead51cc410d05655596e30f281ed9071"
  revision = "04cdfd42973bb9c8589fd6a731800cf222fde1a9"
  revision = "ab9f90f16caab828afda479fd34bfbbbba2efcee"
  revision = "901cec093d58ca36652e8d7906e4bd3e32793801"
  revision = "d8ed2627bdf02c080bf22230dbb337003b7aba2d"
  revision = "18a02ba4a312f95da08ff4cfc0055750ce50ae9e"
  revision = "3b8db5e93c4c02efbc313e17b2e796b0914a01fb"
  revision = "12592ca48efc73057c0edcdddc2a279b37c2b27b"
  revision = "4971afdc2f162e82d185353533d3cf16188a9f4e"
```

Please read the CLA carefully before submitting your contribution to Mercari.
Under any circumstances, by submitting your contribution, you are deemed to accept and agree to be bound by the terms and conditions of the CLA.

https://www.mercari.com/cla/

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+[[constraint]]
+  name = "github.com/BurntSushi/toml"
+  version = "0.2.0"
+
+[[constraint]]
+  name = "github.com/RobotsAndPencils/buford"
+  version = "0.12.0"
+
+[[constraint]]
+  name = "github.com/client9/reopen"
```
Comment (oinume) [inline Gopkg.toml]: It's good to write branch, version or revision to omit following warning from latest dep.

```
dep: WARNING: branch, version, revision, or source should be provided for "github.com/client9/reopen"
```

Comment (jackchuka) [reply Gopkg.toml]: I'll update them to include revisions. Thanks for the advise.

**Comment by cubicdaiya** [comment]
We are still considering about vgo also.

**Comment by cubicdaiya** [comment]
Since as we will switch to Go Modules in the future, we'll close this issue. Thanks.

**Pull request closed without merging**